.gitmodules

```diff
@@ -22,3 +22,7 @@
 [submodule "3rdparty/googletest"]
 	path = 3rdparty/googletest
 	url = https://github.com/google/googletest.git
+[submodule "3rdparty/mkldnn"]
+	path = 3rdparty/mkldnn
+	url = https://github.com/01org/mkl-dnn.git
```

Comment:
FYI, MKL-DNN moved to https://github.com/intel/mkl-dnn and 01org will be redirected but will continue to work.
It's better to update the link.